YAML/powershell_test_bot.yaml

```diff
@@ -0,0 +1,45 @@
+ ## ID - auto generated or manual alpha numeric, must be unique across RLC
+id: powershell_test_bot
+
+## Name - human readable name (e.g., Install MongoDB, Change JIRA password...)
+name: powershell test bot
```

Comment:
Since the name is visible to the end user it is best to give something that clearly conveys the intent. So "Test BOT Using PowerShell Script" is a better name. Also please capitalize the starting letter of each word.